api_proxy.yaml

```diff
@@ -242,27 +242,114 @@ paths:
           description: There was an error in retrieving the user session data.
           schema:
             $ref: "#/definitions/ProblemJson"
+  "/payment-requests/{rptId}":
+    x-swagger-router-controller: PagoPAProxyController
+    parameters:
+      - name: rptId
+        in: path
+        required: true
+        description: Unique identifier for payments.
+        type: string
+    get:
+      operationId: getPaymentInfo
+      summary: Get Payment Info
+      description: Retrieve information about a payment
+      responses:
+        '200':
+          description: Payment information retrieved
+          schema:
+            "$ref": "#/definitions/PaymentRequestsGetResponse"
+          examples:
+            application/json:
+              importoSingoloVersamento: 200,
+              codiceContestoPagamento: "ABC123"
+        "400":
+          description: Invalid input or PagoPA response
+        "500":
+          description: PagoPA services are not available or request is rejected
+  "/payment-activations":
+    x-swagger-drouter-controller: PagoPAProxyController
```

Comment:
typo? `drouter`

Reply:
good catch! Is a copy&paste from https://github.com/teamdigitale/italia-pagopa-proxy/blob/skeleton_with_soap_server/api_pagopa.yaml#L32, I'll comment the relative PR